ethereum/api/src/test/resources/org/hyperledger/besu/ethereum/api/jsonrpc/debug/trace-call/debug_traceCall_all.json

```diff
@@ -28,271 +28,163 @@
         "op" : "PUSH1",
         "gas" : 16755910,
         "gasCost" : 3,
-        "depth" : 1,
-        "stack" : null,
-        "memory" : null,
-        "storage" : null,
-        "reason" : null
+        "depth" : 1
```

Comment:
this seems to have been an issue before, but given the name wouldn't we expect stack, memory, and storage in this trace?  I get why this is as it is, but the name is misleading

Reply:
Yes, I agree the name is a bit weird. I guess "all" here means that all the fields are disabled as the request parameters looks like
    `  "disableMemory": true, "disableStack": true,  "disableStorage": true`
Where debug_traceCall_disableMemory.json has only
`      "disableMemory":true
`

Reply:
I renamed debug_traceCall_complete.json to debug_traceCall_default.json, as by default now, we're not tracing memory.